lib/hmmlearn/_hmmc.pyx

```diff
@@ -1,5 +1,7 @@
 # cython: language_level=3, boundscheck=False, wraparound=False
-
+"""
+Good Readable Reference: https://www.cs.sjsu.edu/~stamp/RUA/HMM.pdf
```

Comment:
I would rather stick to Rabiner1989, which is well established and I think covers the same algorithm (equations 90+).

Reply:
Updated

Reply:
Also, these should go to the tutorial, which already has some bibliography entries.